collections/harmonized-landsat-sentinel.yaml

```diff
@@ -0,0 +1,526 @@
+    https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: https://services-uswest2.sentinel-hub.com/api
+    Name: Sentinel Hub
+    Role: processor
+    Type: hls
+    Notes: Globally since April 2013
+    Primary: true
+CustomScripts:
+  Title:
+  URL:
+WMTS:
+  - href:
+    layer_id:
+    dimensions:
+      warnings: true
+Configurations:
+  - href:
+    rel:
+    type:
+    title:
+  - href:
+    rel:
+    type:
+    title:
+    sentinelhub:layer_name:
+    sentinelhub:mosaicking_order:
+    sentinelhub:upsampling:
+Extent:
+  spatial:
+    bbox:
+      -
+        - -180
+        - -80
+        - 180
+        - 84
+  temporal:
+    interval:
+      -
+        - '2013-04-01T00:00:00Z'
+        - null
+CubeDimensions:
+  x:
+    type: spatial
+    axis: x
+    extent:
+      - -180
+      - 180
+    reference_system: 
+      $schema: https://proj.org/schemas/v0.4/projjson.schema.json
+      type: GeodeticCRS
+      name: AUTO 42001 (Universal Transverse Mercator)
+      datum:
+        type: GeodeticReferenceFrame
+        name: World Geodetic System 1984
+        ellipsoid:
+          name: WGS 84
+          semi_major_axis: 6378137
+          inverse_flattening: 298.257223563
+      coordinate_system:
+        subtype: ellipsoidal
+        axis:
+          - name: Geodetic latitude
+            abbreviation: Lat
+            direction: north
+            unit: degree
+          - name: Geodetic longitude
+            abbreviation: Lon
+            direction: east
+            unit: degree
+      area: World
+      bbox:
+        south_latitude: -80
+        west_longitude: -180
+        north_latitude: 84
+        east_longitude: 180
+      id:
+        authority: OGC
+        version: "1.3"
+        code: Auto42001
+  y:
+    type: spatial
+    axis: y
+    extent:
+      - 84
+      - -80
+    reference_system:
+      $schema: https://proj.org/schemas/v0.4/projjson.schema.json
+      type: GeodeticCRS
+      name: AUTO 42001 (Universal Transverse Mercator)
+      datum:
+        type: GeodeticReferenceFrame
+        name: World Geodetic System 1984
+        ellipsoid:
+          name: WGS 84
+          semi_major_axis: 6378137
+          inverse_flattening: 298.257223563
+      coordinate_system:
+        subtype: ellipsoidal
+        axis:
+          - name: Geodetic latitude
+            abbreviation: Lat
+            direction: north
+            unit: degree
+          - name: Geodetic longitude
+            abbreviation: Lon
+            direction: east
+            unit: degree
+      area: World
+      bbox:
+        south_latitude: -80
+        west_longitude: -180
+        north_latitude: 84
+        east_longitude: 180
+      id:
+        authority: OGC
+        version: "1.3"
+        code: Auto42001
+  t:
+    type: temporal
+    extent:
+      - '2013-04-01T00:00:00Z'
+      - null
+    step: P5D
```

Comment:
I think this should be `null` as the steps are irregularly

> The space between the temporal instances as [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations), e.g. P1D. Use null for irregularly spaced steps.

https://github.com/stac-extensions/datacube#temporal-dimension-object

Reply:
I agree, changed to `null`.